frontend/src/cljs/imas_seamap/events.cljs

```diff
@@ -232,6 +233,11 @@
                    :uri             meow-boundaries-url
                    :response-format (ajax/json-response-format {:keywords? true})
                    :on-success      [:sok/update-meow-boundaries]
+                   :on-failure      [:ajax/default-err-handler]}
+                  {:method          :get
+                   :uri             story-maps-url
+                   :response-format (ajax/json-response-format {:keywords? true})
+                   :on-success      [:sm/update-featured-maps]
```

Comment:
Might be worth thinking about this; the main question is whether we load on-demand or at startup I guess.  Not really that much difference I suppose.

Reply:
I chose to do it this way so that it's easy to hydrate the currently selected feature map from it's ID (if one currently is selected) when we load the state.